packages/components/forms/src/Label/Label.tsx

```diff
@@ -0,0 +1,66 @@
+import { cx, css } from 'emotion';
+import React, { forwardRef } from 'react';
+import type { HTMLProps } from 'react';
+import tokens from '@contentful/f36-tokens';
+import { Box } from '@contentful/f36-core';
+import type { BoxProps } from '@contentful/f36-core';
+
+export type LabelInternalProps = {
+  children: React.ReactText;
+  htmlFor: HTMLProps<HTMLLabelElement>['htmlFor'];
```

Comment:
👍🏼 for requiring

Reply:
Can we please add some docs `Add the ID for the input this label is for` or something?